Mathlib/SetTheory/Game/Impartial.lean

```diff
@@ -26,11 +26,11 @@ namespace PGame
 
 /-- The definition for an impartial game, defined using Conway induction. -/
 def ImpartialAux (G : PGame) : Prop :=
-  (G ≈ -G) ∧ (∀ i, ImpartialAux (G.moveLeft i)) ∧ ∀ j, ImpartialAux (G.moveRight j)
+  (G ≡ -G) ∧ (∀ i, ImpartialAux (G.moveLeft i)) ∧ ∀ j, ImpartialAux (G.moveRight j)
```

Comment:
As @vihdzp pointed out, moving the definition to use `Identical` means it won't hold for games considered impartial in the literature (i.e. `{∗1 + ∗2 | ∗3}`).

Reply:
The issue is a bit more subtle actually; the literature does require `G ≡ -G`, but we don't! This makes our results ever so slightly more general. I'm not aware of any important results that would only hold for the stronger version of `Impartial`.

Reply:
I'll take some more control of this PR and remove this change :+1: ([this change was discussed on zulip](https://leanprover.zulipchat.com/#narrow/channel/116395-maths/topic/Poset.20games/near/495609460))